packages/valaxy-theme-press/components/nav/PressNavBar.vue

```diff
@@ -35,9 +35,13 @@ const themeConfig = useThemeConfig()
 
       <PressToggleLocale m="x-2" />
 
-      <button m="x-2" type="button" aria-label="Toggle Dark Mode" @click="toggleDark()">
-        <div v-if="!isDark" i-ri-sun-line />
-        <div v-else i-ri-moon-line />
+      <button class="switch switchAppearance" type="button" aria-label="Toggle Dark Mode" @click="toggleDark()">
```

Comment:
I think you should separate it into a separate component.

`<PressSwitchAppearance />`

Reply:
good idea, thanks